libcob/coblocal.h

```diff
@@ -22,6 +22,8 @@
 #ifndef COB_LOCAL_H
 #define COB_LOCAL_H
 
+#include "config.h"
```

Comment:
@GitMensch Any reason why `config.h` is not included systematically in all other `.h` files ?